JobScheduler/ApiRequestHandler.cs

```diff
@@ -65,7 +65,7 @@ public ApiRequestHandler(IHttpClientFactory factory, Config config)
         public async Task<List<UserTask>> GetExpectedGraduationUsersAsync()
         {
             HttpResponseMessage response =
-                await client.GetAsync("api/UserTask/CreateUserTasks/" + config.JobSchedulerConfig.TimeRange);
+                await client.GetAsync("localhost/UserTask/CreateUserTasks/" + config.JobSchedulerConfig.TimeRange);
```

Comment:
Why did you change this? I guess it should be api/UserTask/CreateUserTasks/ since it is communicating with the api.

The base url is set in the program of this application.

Reply:
I do not remember making this change at all. I will change it back.